traffic_portal/app/src/common/api/AuthService.js

```diff
@@ -57,6 +57,25 @@ var AuthService = function($rootScope, $http, $state, $location, $q, $state, htt
         return deferred.promise;
     };
 
+    this.oauthLogin = function(token) {
+        return httpService.post(ENV.api['root'] + 'user/login/oauth', { t: token})
+            .then(
+                function(result) {
+                    $rootScope.$broadcast('authService::oauthLogin');
```

Comment:
you can change this to 

`$rootScope.$broadcast('authService::login');`

if you do a search for authService::login you'll see what dispatching that event does.

Reply:
done